Doppler.reportingApi/Controllers/SummaryController.cs

```diff
@@ -0,0 +1,59 @@
+using Doppler.ReportingApi.Models;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Doppler.ReportingApi.Controllers
+{
+    [ApiController]
+    public class SummaryController : ControllerBase
+    {
+        /// <summary>
+        /// Return an object summarizing the campaingns performance of an user
+        /// </summary>
+        /// <param name="accountName">User name</param>
+        [HttpGet]
+        [Route("{accountName}/summary/campaigns")]
+        [ProducesResponseType(typeof(CampaignsSummary), 200)]
+        [Produces("application/json")]
+        public async Task<IActionResult> Campaigns(string accountName, [FromQuery] BasicDatefilter dateFilter)
+        {
```

Comment:
Could you include some tests for this functionality?

Reply:
yes :)